main.go

```diff
@@ -19,6 +25,14 @@ import (
 	"strings"
 	"sync"
 	"time"
+
```

Comment:
I'm going to have to look into whether I can get my IDE to separate these imports out. I'm assuming the distinction is between official go and third party imports?